src/routes/index.tsx

```diff
@@ -1,10 +1,13 @@
 import { VNode } from "preact";
+import DailyAuthsReport from "../daily-auths-report";
+import DailyDropffsReport from "../daily-dropoffs-report";
 import { Router } from "../router";
 import HomeRoute from "./home-route";
 import ReportRoute from "./report-route";
```

Comment:
I'd probably rename this to reflect the fact it's now a higher-order function, not a component.

e.g. `createReportRoute` or `createReportRouteComponent`

Reply:
done f4eba939cc5d4caa9f6c2b3da487401da29b6020